### PR TITLE
Feature/objects 1189 landing page

### PR DESCRIPTION
**Pull request by andreassiegel**
### What changes were proposed in this pull request?

This adds a DevKit landing page. It is based on the [smart-comos.com](https://www.smart-cosmos.com) webpage.

The content is mostly copied from the README.

### CODE REVIEW

**Review by sdomula — APPROVED** [review]

File: html/new/index.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
```
Comment (wfschneider-rfid) [inline html/new/index.html]: Why is this file here and an exact copy of `./index.html`

File: start [hunk 2]
```diff
@@ -80,7 +80,7 @@ function prerequisites() {
   docker-compose pull
 
   echo -e "\n${ORANGE}→ Starting database and message queue services...${DEFAULT_COLOR}"
-  docker-compose up -d mysql zookeeper kafka
+  docker-compose up -d mysql zookeeper kafka landing-page
```
Comment (wfschneider-rfid) [inline start]: I think the landing page needs to be the last thing started.

**Review by ghost — CHANGES_REQUESTED** [review]

File: html/new/index.html (hunk 1 above)
Comment (andreassiegel) [inline html/new/index.html]: Atom screwed things up on save. :/

File: start (hunk 2 above)
Comment (andreassiegel) [inline start]: It actually doesn't matter. `nginx` starts in basically no time, and in the end all depends on the gateway.

**Review by ghost — APPROVED** [review]
👍
